WORKSPACE

```diff
@@ -338,6 +428,11 @@ maven_jar(
   artifact = "com.101tec:zkclient:0.3"
 )
 
+#http_jar(
```

Comment:
The Dhalion build files and the dependencies have been temporarily disabled till maven repo is enabled. I would suggest we keep it like this for a few more days.